Softeq.XToolkit.PushNotifications.iOS/Softeq.XToolkit.PushNotifications.iOS.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0-ios10.0</TargetFramework>
+    <TargetFramework>net8.0-ios12.0</TargetFramework>
```

Comment:
Can we use iOS 17?

Reply:
We should use a minimal target framework of only used APIs to support all iOS 12+ targets as much as possible.

For some projects that used Maui Essentials ios17 target is required (Permissions etc.)